public/javascripts/control.js

```diff
@@ -836,9 +840,62 @@
+                          validation: [ 'required', 'xmlLegalChars', 'unique' ],
+                          required: true,
+                          value: 'audit',
+                          summary: false },
+            location_priority: {
+                          name: 'Location Priority',
+                          type: 'enum',
+                          description: 'This setting balances location accuracy with battery life.',
+                          tips: [
+                              'See <a href="https://developers.google.com/android/reference/com/google/android/gms/location/LocationRequest" ' +
+                              'target="_" rel="nofollow">the LocationRequest documentation</a> for an explanation of the options. ' +
+                              'Using "balanced" will capture the location at least at specified intervals while preserving power. ' +
+                              'Using "off" will disable location audit altogether.'],
+                          options: ['off', 'no-power', 'low-power', 'balanced', 'high-accuracy'],
+                          value: 'balanced',
+                          summary: false },
+            location_min_interval: {
+                          name: 'Location Minimum Interval',
+                          type: 'text',
+                          description: 'This setting governs how often location is captured for the audit log.',
+                          tips: [
+                              'The desired minimum time, in seconds, location updates will be fetched. ' +
+                              'Required to enable location in audit log. To disable location audit, select "off" in the priority.' +
+                              'Default: 20 seconds.'
+                          ],
+                          value: '20',
+                          summary: false },
+            location_max_age: {
+                          name: 'Location Maximum Age',
+                          type: 'text',
+                          description: 'The maximum time, in seconds, locations will be considered valid.',
+                          tips: [
+                              'Must be greater than or equal to the minimum interval. ' +
+                              'Required to enable location in audit log. To disable location audit, select "off" in the priority. ' +
+                              'Default: 60 seconds (1 minute).'
+                          ],
+                          value: '60',
+                          summary: false },
+            track_changes: {
+                          name: 'Track Changes',
+                          type: 'enum',
+                          description: 'Whether to track changes to the form made any time before submission.',
+                          tips: [],
+                          options: ['true', 'false'],
+                          value: 'true',
+                          summary: false },
+          },
```

Comment:
Audit gets a custom set of control properties: no `kind` (as there's only one choice, "audit"), plus the four metadata attributes.